src/main/java/marquez/api/Namespace.java

```diff
@@ -1,64 +1,24 @@
 package marquez.api;
```

Comment:
There's an open issue to make this a NamespaceResponse and use it for creates and gets (and delete GetNamespaceResponse).